components/aws/sagemaker/TrainingJob/component.yaml

```diff
@@ -179,11 +190,16 @@ outputs:
       type: String,
       description: "The status of the training job.",
     }
+  - {
+      name: warm_pool_status,
+      type: JsonObject,
+      description: "The status of the warm pool associated with the training job.",
+    }
   ###########################GENERATED SECTION ABOVE############################
 
 implementation:
   container:
-    image: public.ecr.aws/kubeflow-on-aws/aws-sagemaker-kfp-components:2.0.0
```

Comment:
make sure to change this before merging

Reply:
Yea

Reply:
release version should be 2.1.0, last digit is incremented for bug fixes https://semver.org/